### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by Gorkowski**
As I'll be doing a lot here

### CODE REVIEW

**Comment by Gorkowski** [comment]
@ngam, or you can change this repo to the particula format of multiple reviewers (this fix was the first I found)

File: .github/codeowners
Comment (ngam) [inline .github/codeowners:1]: /

Comment (ngam) [reply .github/codeowners:1]: Can we try @Gorkowski (with a capital letter)?

**Comment by ngam** [comment]
Why is it showing an error??

**Comment by ngam** [comment]
For some weird reason, you didn’t have access to write into this repo. I thought we had you as admin of the entire organization… I will double check later.

**Pull request merged**